### PR TITLE
Issue 49 summarize changes

### PR DESCRIPTION
**Pull request by tparchambault**
Extended a standard Gtk Dialog to encapsulate a list of the changes to be deployed.

### CODE REVIEW

**Comment by tparchambault** [comment]
This may be the issue: https://github.com/rust-lang/rust/issues/81654

File: python/fapolicy_analyzer/locale/fapolicy_analyzer.pot [hunk 1]
```diff
@@ -7,9 +7,9 @@
 #, fuzzy
 msgid ""
 msgstr ""
-"Project-Id-Version: fapolicy-analyzer 0.0.4\n"
+"Project-Id-Version: fapolicy-analyzer snapshot\n"
```
Comment (jw3) [inline python/fapolicy_analyzer/locale/fapolicy_analyzer.pot:10]: Wonder if this is being updated when an RPM build is performed...

Comment (tparchambault) [reply python/fapolicy_analyzer/locale/fapolicy_analyzer.pot:10]: If that's the case, wouldn't we see the issue if we installed the newly created rpm on build system? I can try it then rollback the system.

File: python/fapolicy_analyzer/ui/confirm_changes_dialog.py [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+
+
+def main():
+    win = DialogWindow()
+    win.connect("destroy", Gtk.main_quit)
+    win.show_all()
+    Gtk.main()
+
+
+if __name__ == "__main__":
```
Comment (dorschs57) [inline python/fapolicy_analyzer/ui/confirm_changes_dialog.py:82]: I don't think we want any example or testing code to be committed with new component.

File: python/fapolicy_analyzer/locale/fapolicy_analyzer.pot (hunk 1 above)
Comment (jw3) [reply python/fapolicy_analyzer/locale/fapolicy_analyzer.pot:10]: Will verify this in #148 

Non-blocking for this PR

**Comment by dorschs57** [comment]
I don't think we are using the ConfirmationDialog component any more then, so it can be deleted (py, glade, test files, and possibly string file entries).

**Comment by dorschs57** [comment]
Also we need some test coverage on the new ConfirmationChangesDialog.  You should be able to use the old test_confirmation_dialog.py file as a starting point.

File: python/fapolicy_analyzer/ui/confirm_changes_dialog.py [hunk 3]
```diff
@@ -0,0 +1,83 @@
+import gi
+gi.require_version("Gtk", "3.0")
+from gi.repository import Gtk
+import ui.strings as strings
+
+
+class ConfirmInfoDialog(Gtk.Dialog):
```
Comment (dorschs57) [inline python/fapolicy_analyzer/ui/confirm_changes_dialog.py:7]: For consistency the class name should correspond with the python file name.

File: python/fapolicy_analyzer/ui/confirm_changes_dialog.py (hunk 2 above)
Comment (tparchambault) [reply python/fapolicy_analyzer/ui/confirm_changes_dialog.py:82]: Done.

File: python/fapolicy_analyzer/ui/confirm_changes_dialog.py (hunk 3 above)
Comment (tparchambault) [reply python/fapolicy_analyzer/ui/confirm_changes_dialog.py:7]: Done. Will push shortly.

**Pull request merged**